docs/source/basics.rst

```diff
@@ -3,202 +3,147 @@
 Programs and Gates
 ==================
 
-First, initialize a localQVM instance on your laptop. You should have two consoles open in your terminal to run in the
-background.
+.. note::
 
-.. code:: python
-
-    ### CONSOLE 1
-    $ quilc -S
-    port triggered: 6000.
-    [2018-09-19 11:22:37] Starting server: 0.0.0.0 : 6000.
+    If you're running locally, remember set up the QVM and quilc in server mode before trying to use
+    them: :ref:`server`.
 
-    ### CONSOLE 2
-    $ qvm -S
-    Welcome to the Rigetti QVM
-    (Configured with 2048 MiB of workspace and 8 workers.)
-    [2018-09-20 15:39:50] Starting server on port 5000.
+Introduction
+~~~~~~~~~~~~
 
-Quantum programs are written in Forest using the ``Program`` object from the ``quil`` module.
+Quantum programs are written in Forest using the ``Program`` object. This ``Program`` abstraction will help us
+compose `Quil programs <https://arxiv.org/abs/1608.03355>`_.
 
 .. code:: python
 
-    from pyquil import Program, get_qc
-    from pyquil.gates import *
+    from pyquil import Program
 
-Programs are then constructed from quantum gates, which can be found in the ``gates`` module. We can
-add quantum gates to programs in numerous ways, including using the ``.inst(...)`` method. We use
-the ``.measure(...)`` method to measure qubits into classical registers:
+Programs are constructed by adding quantum gates to it, which are defined in the ``gates`` module. We can import all
+standard gates with the following:
 
 .. code:: python
 
-    p = Program()
-    p.inst(X(0)).measure(0, 0)
-
-.. parsed-literal::
+     from pyquil.gates import *
 
-    <pyquil.quil.Program at 0x101d45a90>
+Let's instantiate a ``Program`` and add an operation to it. We will act an ``X`` gate on qubit 0.
 
+.. code:: python
 
-This program simply applies the :math:`X`-gate to the zeroth qubit,
-measures that qubit, and stores the measurement result in the zeroth
-classical register. We can look at the Quil code that makes up this
-program simply by printing it.
+    p = Program()
+    p += X(0)
 
-.. code:: python
+All qubits begin in the ground state. This means that if we measure a qubit without applying operations on it, we expect to receive
+a 0 result. The ``X`` gate will rotate qubit 0 from the ground state to the excited state, so a measurement immediately
+after should return a 1 result. More details about gate operations are explained in :ref:`intro`.
 
-    print(p)
+We can print our pyQuil program (``print(p)``) to see the equivalent Quil representation:
 
 .. parsed-literal::
 
     X 0
```

Comment:
This might be able to be made more clear at first glance as program output. I'm not overly familiar w/ Sphinx but do you think we could leverage the `program-output` to make this a bit more clear? https://pythonhosted.org/sphinxcontrib-programoutput/ Not super important though